guides/v2.1/cloud/project/new-relic.md

```diff
@@ -33,26 +67,41 @@ runtime:
 
 ## Add New Relic APM to an environment
 
-The {{site.data.var.ece}} plans to support up to three instances of your New Relic APM license across all environments. We recommend adding a New Relic license to your Staging and Production environments, and you can add the license to one other environment of your choice.
+The {{site.data.var.ece}} plans support up to three instances of your New Relic APM license across all environments. We recommend adding a New Relic license to your Pro Staging and Production environments, and you can add the license to one other environment of your choice. 
+
+{:.bs-callout .bs-callout-info}
+If you have more than three active environments using the same New Relic license key, you need to remove a license variable from an existing environment.
+Contact your Magento technical account manager or New Relic sales (sales@newrelic.com) if you want to use New Relic on more than three environments.
```

Comment:
It says three environments here, but check if you have to specify "active". I think the license can be present on unlimited inactive, it's just the active environments that make a difference.